### PR TITLE
Remove node-canvas devDependency

### PR DESCRIPTION
**Pull request by t3chguy**
Requires https://github.com/matrix-org/matrix-react-sdk/pull/6358

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -168,7 +167,8 @@
       "decoderWorker\\.min\\.js": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js",
       "decoderWorker\\.min\\.wasm": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js",
       "waveWorker\\.min\\.js": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js",
-      "context-filter-polyfill": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js"
+      "context-filter-polyfill": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js",
+      "FontManager.ts": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/FontManager.js"
```
Comment (ara4n) [inline package.json:171]: .ts shirley?

Comment (ara4n) [reply package.json:171]: or hsa it been compiled down to .js by this point?

Comment (ara4n) [reply package.json:171]: oh, no the mock really is .js, even though it's mocking a .ts file. nm.

**Review by ara4n — APPROVED** [review]
lgtm

File: package.json (hunk 1 above)
Comment (t3chguy) [reply package.json:171]: Yeah whilst `ts` probably would have worked, files outside `src` don't count on our graphs anyway. I fear Jest would need some coercing to be happy with TS there though did not bother trying (rest of mocks are JS too)

**Pull request merged**